rna-germline-variant-calling.wdl

```diff
@@ -0,0 +1,777 @@
+		docker: "broadinstitute/genomes-in-the-cloud:2.3.1-1504795437"
+		memory: "4 GB"
+    	preemptible: preemptible_count
+    }
+}
+
+
+task BaseRecalibrator {
+
+    File input_bam
+    File input_bam_index
+    String recal_output_file
+
+    File dbSNP_vcf
+    File dbSNP_vcf_index
+    Array[File] known_indels_sites_VCFs
+    Array[File] known_indels_sites_indices
+
+    File ref_dict
+    File ref_fasta
+    File ref_fasta_index
+
+    Int preemptible_count
+
+    command <<<
+        /gatk/gatk-launch --javaOptions "-XX:GCTimeLimit=50 -XX:GCHeapFreeLimit=10 -XX:+PrintFlagsFinal \
+            -XX:+PrintGCTimeStamps -XX:+PrintGCDateStamps -XX:+PrintGCDetails \
+            -Xloggc:gc_log.log -Xms4000m" \
+            BaseRecalibrator \
+            -R ${ref_fasta} \
+            -I ${input_bam} \
+            --useOriginalQualities \
+            -O ${recal_output_file} \
+            -knownSites ${dbSNP_vcf} \
+            -knownSites ${sep=" -knownSites " known_indels_sites_VCFs}
+    >>>
+
+    output {
+        File recalibration_report = recal_output_file
+    }
+
+    runtime {
+        memory: "6 GB"
+        disks: "local-disk " + sub((size(input_bam,"GB")*3)+30, "\\..*", "") + " HDD"
+        docker: "broadinstitute/gatk:4.beta.6"
+        preemptible: preemptible_count
+    }
+}
+
+
+task ApplyBQSR {
+
+    File input_bam
+    File input_bam_index
+    String base_name
+    File recalibration_report
+
+    File ref_dict
+    File ref_fasta
+    File ref_fasta_index
+    #Is there a good default for compression level? just remove the -D arg completely if no preference is provided by the user?
+    Int? compression_level
+    Int preemptible_count
+
+    command <<<
+        /gatk/gatk-launch --javaOptions "-XX:+PrintFlagsFinal -XX:+PrintGCTimeStamps -XX:+PrintGCDateStamps \
+            -XX:+PrintGCDetails -Xloggc:gc_log.log \
+            -XX:GCTimeLimit=50 -XX:GCHeapFreeLimit=10 -Dsamjdk.compression_level=${default=2 compression_level} -Xms3000m" \
+            ApplyBQSR \
+            --addOutputSAMProgramRecord \
+            -R ${ref_fasta} \
+            -I ${input_bam} \
+            --useOriginalQualities \
+            -O ${base_name}.bam \
+            -bqsr ${recalibration_report} \
+            --EOQ true
+    >>>
+
+    output {
+        File output_bam = "${base_name}.bam"
+        File output_bam_index = "${base_name}.bai"
+    }
+
+    runtime {
+        memory: "3500 MB"
+        disks: "local-disk " + sub((size(input_bam,"GB")*4)+30, "\\..*", "") + " HDD"
+        preemptible: preemptible_count
+        docker: "broadinstitute/gatk:4.beta.6"
+    }
+}
+
+task HaplotypeCaller {
+
+	File input_bam
+	File input_bam_index
+	String base_name
+
+	File interval_list
+
+  	File ref_dict
+  	File ref_fasta
+  	File ref_fasta_index
+
+  	Int preemptible_count
+
+	command <<<
+		java -jar /usr/gitc/GATK35.jar \
+		    -T HaplotypeCaller \
+		    -R ${ref_fasta} \
+		    -I ${input_bam} \
+		    -ERC GVCF \
```

Comment:
remove above line

Reply:
re-running without GenotypeGVCFs in FireCloud now.